.github/workflows/latest-matrix-sdk-crypto.yml

```diff
@@ -0,0 +1,87 @@
+name: Build against latest matrix-sdk-crypto
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '22 1 * * *'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  test-against-latest-matrix-sdk-crypto:
+    name: test-against-latest-matrix-sdk-crypto
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Load cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          save-if: ${{ github.ref == 'refs/heads/main' }}
+
+      - name: Update to the latest matrix-sdk-crypto version
+        run: cargo update matrix-sdk-crypto
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.0
+
+      - name: Install yarn
+        run: npm install --global yarn
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build the WebAssembly + JavaScript binding
+        run: yarn build
+
+      - name: Test the JavaScript binding
+        run: yarn test
+
+      - name: Open an issue on failure
+        if: failure()
+        uses: JasonEtco/create-an-issue@1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5 # v2.9.2
```

Comment:
They haven't created a Github tag? Otherwise I'm pretty sure it's possible to use the tag name after the `@` symbol.

Reply:
It's best practice to use a commit hash for action version to avoid a malicious action owner being able to execute arbitrary code within our build. Ref: https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Ah, good point, TIL, thanks!

Reply:
In case anyone is reading along, I realised I missed a step in the explanation - if we use a tag, the action owner (or someone who has taken over their account) can modify a tag, making it point at a different commit, whereas the commit id guarantees* we are getting the code we think we're getting.

 * except, of course, if the attacker cleverly generates a malicious commit with the same commit id, which is difficult but not impossible with SHA1.